Signal/src/Models/TSMessageAdapaters/TSMessageAdapter.m

```diff
@@ -177,19 +177,9 @@ - (instancetype)initWithInteraction:(TSInteraction *)interaction
                     }
                 } else if ([attachment isKindOfClass:[TSAttachmentPointer class]]) {
                     TSAttachmentPointer *pointer = (TSAttachmentPointer *)attachment;
-                    adapter.messageType          = TSInfoMessageAdapter;
-
-                    switch (pointer.state) {
-                        case TSAttachmentPointerStateEnqueued:
-                            adapter.messageBody = NSLocalizedString(@"ATTACHMENT_QUEUED", nil);
```

Comment:
They're all *started* from a serial queue. But then they quickly get dispatched to a concurrent network queue. So no limit AFAIK.